src/v2/Apps/Fair/routes.tsx

```diff
@@ -22,19 +24,49 @@ export const routes: RouteConfig[] = [
     `,
     children: [
       {
-        path: "/",
-        getComponent: () => FairOverviewRoute,
+        path: "",
+        getComponent: () => FairExhibitorsRoute,
         prepare: () => {
-          FairOverviewRoute.preload()
+          FairExhibitorsRoute.preload()
         },
         query: graphql`
-          query routes_FairOverviewQuery($slug: String!) {
+          query routes_FairExhibitorsQuery($slug: String!) {
             fair(id: $slug) {
-              ...FairOverview_fair
+              ...FairExhibitors_fair
             }
           }
         `,
       },
+      {
+        path: "artworks",
+        getComponent: () => FairArtworksRoute,
+        prepare: () => {
+          FairArtworksRoute.preload()
+        },
+        query: graphql`
+          query routes_FairArtworksQuery($slug: String!) {
+            fair(id: $slug) {
+              ...FairArtworks_fair
+            }
+          }
+        `,
+      },
+    ],
+  },
+  {
+    path: "/fair2/:slug",
```

Comment:
Oh interesting-- so even though this has the same path as above, I'm guessing when we fall through the `/fair2/:slug` naked route it'll get caught by the one above?

This seems a bit cleaner than before.

Reply:
Yeah, it just matches first

![](http://static.damonzucconi.com/_capture/VjbLHOAumAnQ.gif)

Reply:
This is interesting but also a little confusing; a comment in the code would be good